sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/CreateComposedModel.java

```diff
@@ -0,0 +1,94 @@
+        // Instantiate a source client which has the model that we want to copy.
+        FormTrainingClient client = new FormTrainingClientBuilder()
+            .credential(new AzureKeyCredential("{key}"))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildClient();
+
+        // Train custom model
+        String model1TrainingFiles = "{SAS_URL_of_your_container_in_blob_storage_for_model_1}";
+        // The shared access signature (SAS) Url of your Azure Blob Storage container with your forms.
+        SyncPoller<FormRecognizerOperationResult, CustomFormModel> model1Poller = client.beginTraining(model1TrainingFiles, true);
+
+        // Train custom model
+        String model2TrainingFiles = "{SAS_URL_of_your_container_in_blob_storage_for_model_2}";
+        // The shared access signature (SAS) Url of your Azure Blob Storage container with your forms.
+        SyncPoller<FormRecognizerOperationResult, CustomFormModel> model2Poller = client.beginTraining(model2TrainingFiles, true);
+
+        String labeledModelId1 = model1Poller.getFinalResult().getModelId();
+        String labeledModelId2 = model2Poller.getFinalResult().getModelId();
+
+        final CustomFormModel customFormModel
+            = client.beginCreateComposedModel(Arrays.asList(labeledModelId1, labeledModelId2),
+            new CreateComposedModelOptions()
+                .setModelDisplayName("my composed model name")
+                .setPollInterval(Duration.ofSeconds(5)),
+            Context.NONE)
+            .getFinalResult();
+
+        System.out.printf("Model Id: %s%n", customFormModel.getModelId());
+        System.out.printf("Model Status: %s%n", customFormModel.getModelStatus());
+        System.out.printf("Model display name: %s%n", customFormModel.getModelDisplayName());
+        System.out.printf("Is this a composed model: %s%n",
+            customFormModel.getCustomModelProperties().isComposed());
+        System.out.printf("Composed model creation started on: ", customFormModel.getTrainingStartedOn());
+        System.out.printf("Composed model creation completed on: ", customFormModel.getTrainingCompletedOn());
+
+        System.out.println("Recognized Fields:");
+        customFormModel.getSubmodels().forEach(customFormSubmodel -> {
+            System.out.printf("The subModel with form type %s has accuracy: %.2f%n",
+                customFormSubmodel.getFormType(), customFormSubmodel.getAccuracy());
+            customFormSubmodel.getFields().forEach((label, customFormModelField) ->
+                System.out.printf("The model found field '%s' to have name: %s with an accuracy: %.2f%n",
+                    label, customFormModelField.getName(), customFormModelField.getAccuracy()));
+        });
+        System.out.println();
+
+        customFormModel.getTrainingDocuments().forEach(trainingDocumentInfo -> {
+            System.out.printf("Document name: %s%n", trainingDocumentInfo.getName());
```

Comment:
possibly include the training document's modelID here since it is relevant for composed model